lib/pdf_fill/forms/va21p530v2.rb

```diff
@@ -566,13 +572,20 @@ def expand_tours_of_duty(tours_of_duty)
         end
       end
 
+      # rubocop:disable Layout/LineLength
       def convert_location_of_death
         location_of_death = @form_data['locationOfDeath']
         return if location_of_death.blank?
 
+        if location_of_death[location_of_death['location']].present? && location_of_death['location'] != 'other'
+          location_of_death['placeAndLocation'] = "#{location_of_death[location_of_death['location']]['facilityName']} - #{location_of_death[location_of_death['location']]['facilityLocation']}"
```

Comment:
`rubocop:disable Layout/LineLength` is not appropriate in this situation. Please break up line 581 into multiple lines